tofhir-server/src/test/scala/io/tofhir/server/project/ProjectEndpointTest.scala

```diff
@@ -112,4 +112,46 @@ class ProjectEndpointTest extends BaseEndpointTest {
       }
     }
   }
+
+  "General rejections" should {
+
+    "Try to create a project without application/json content type" in {
+      // Send a POST request without changing the contentType to application/json
+      Post(s"/${webServerConfig.baseUri}/projects", akka.http.scaladsl.model.HttpEntity.apply(writePretty(project1))) ~> route ~> check {
+        status shouldEqual StatusCodes.UnsupportedMediaType
+        val response = responseAs[String]
+        response should include("Type: https://tofhir.io/errors/UnsupportedMediaType")
```

Comment:
Can't you parse the response into a well-defined object. Isn't it a ToFhirError?

Reply:
No, I cannot. The response itself is a string not an object.
Ex:  ![image](https://github.com/srdc/tofhir/assets/123316566/fc8731de-f414-4445-a30f-97c1281d9937)

Reply:
Because, you are returning string representation of the error from the server. See my comment above. Also, after deserializing the error into ToFhirError update the test assertions accordingly e.g. error.status or error.detail, etc

Reply:
I was mistaken. Currently, the error responses are HttpErrorResponses with a field that includes string of the accompanying ToFhirError class.  A proper marshaller will be created and this stringifying logic will be changed after marshaller is created.